src/components/templates/new-user/utlis/connect-more-auth-dialog.tsx

```diff
@@ -0,0 +1,141 @@
+import {
+  Avatar,
+  Box,
+  Button,
+  Dialog,
+  DialogContent,
+  Icon,
+  IconButton,
+  Link,
+  Paper,
+  Stack,
+  Typography,
+} from '@mui/material';
+import { FaTwitter } from 'react-icons/fa';
+import CloseIcon from '@mui/icons-material/Close';
+import { SocialAuthCard } from '../../../atoms/social-auth-card';
+import { Dispatch, SetStateAction } from 'react';
+import { AiFillGithub } from 'react-icons/ai';
+import useTranslation from 'next-translate/useTranslation';
+import { ConnectionHandlerGithub } from '@/services/social-connectors/github-connection';
+import { ConnectionHandlerTwitter } from '@/services/social-connectors/twitter-connection';
+
+type Props = {
+  open: boolean;
+  setOpen: Dispatch<SetStateAction<boolean>>;
+};
+
+export function ConnectMoreAuthDialog({ open, setOpen }: Props) {
+  const { t } = useTranslation('dashboard-new-user');
+  const githubConnection = ConnectionHandlerGithub();
+  const twitterConnection = ConnectionHandlerTwitter();
```

Comment:
please, prefix both handlers with `use` (on the function itself at `@/services/social-connectors/x`). 
I've checked both methods and they're are hooks, so we must tell to react that. (and hooks starts with `use`)

```suggestion
  const githubConnection = useConnectionHandlerGithub();
  const twitterConnection = useConnectionHandlerTwitter();
```